### PR TITLE
Report abuse page non ajax

### PR DESCRIPTION
**Pull request by gidsg**
Adds a form at `/discussion/report-abuse/:commentId` to Report Abuse on comments for non AJAX users. This is to replace the form currently provided by D2 microapp e.g. http://discussion.theguardian.com/components/report-abuse/27576482 which we would like to deprecate.

_N.B_ The styling will look a bit uglier that that shown until the changes in https://github.com/guardian/frontend/pull/11447 are merged. I believe @jamespamplin is looking into this.
## Tasks after merging this PR:
- [ ] Ship sister PR https://github.com/guardian/fastly-edge-cache/pull/542 to make routes HTTPS only
- [ ] Add link to new route 

screen shot 2016-01-08 at 12 17 22
screen shot 2016-01-08 at 12 17 07
screen shot 2016-01-08 at 12 27 03

CC @guardian/discussion @adamnfish

### CODE REVIEW

**Comment by paperboyo** [comment]
"Email" field says it's optional. Why does it give "Please enter a valid email address" warning when empty?

**Comment by gidsg** [comment]
Yeah @paperboyo, it is optional but if the user enters and invalid email it comes back without their original input. I will see if I can fix this.

**Comment by paperboyo** [comment]
Thanks, @gidsg! I took into account that it may warn only on invalid while allowing empty. Haven't thought that empty may mean "I just forgot what you've put here"...

**Comment by gidsg** [comment]
OK @paperboyo I've updated it now to retain the user input in the case of an invalid email address.
screen shot 2016-01-08 at 15 25 03

**Comment by paperboyo** [comment]
Thank you, @gidsg!

**Comment by gidsg** [comment]
Please may someone from the .com health team review this? CC @rich-nguyen

File: discussion/app/views/fragments/reportCommentForm.scala.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
+@(commentId: Int, userForm: Form[discussion.model.DiscussionAbuseReport], errorMessage: Option[String] = None)(implicit request: RequestHeader)
+@if(errorMessage) {
+    <div class="form-field--error">
+        <label class="label">@errorMessage.get</label>
+    </div>
+}
+@defining(userForm.apply("email").value) { emailUserInput =>
+    @defining(userForm.error("categoryId")) { categoryIdError =>
+        @defining(userForm.error("email")) { emailError =>
+            @defining(userForm.error("reason")) { reasonError =>
```
Comment (rich-nguyen) [inline discussion/app/views/fragments/reportCommentForm.scala.html]: can you define a tuple instead?

File: discussion/conf/application.conf [hunk 2]
```diff
@@ -35,8 +35,14 @@ play {
...
-
-  ws {
+  filters {
+      csrf {
+          cookie.name = GU_CSRF
+          cookie.secure = true
+          sign.tokens = true
+      }
+  }
+    ws {
```
Comment (rich-nguyen) [inline discussion/conf/application.conf]: indentation

**Comment by rich-nguyen** [comment]
:+1:

File: discussion/app/views/fragments/reportCommentForm.scala.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    <div class="form-field--error">
+        <label class="label">@errorMessage.get</label>
+    </div>
+}
+@defining(
+    userForm.apply("email").value,
+    userForm.error("categoryId"),
+    userForm.error("email"),
+    userForm.error("reason")
+) { formValidationErrors =>
```
Comment (rich-nguyen) [inline discussion/app/views/fragments/reportCommentForm.scala.html]: I mean something like `(email, categoryError, emailError, reason)`

**Pull request merged**